app/services/consult_service.py

```
@@ -473,4 +479,180 @@ def delete_consult_session(
+            user_id=user_id,
+            session_id=session_id,
+            summary=summary_text,
+        )
+        db.add(summary_row)
+        db.commit()
+        db.refresh(summary_row)
+
+        return summary_row.summary
+    except Exception as e:
+        db.rollback()
+        logger.error(f"[{session_id}] 요약 저장 실패: {e}", exe_info=True)
```

Ruff (0.14.7)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)